guides/micronaut-openapi-generator-server/java/docs/models/BookInfo.md

```diff
@@ -0,0 +1,22 @@
+
+
+# BookInfo
```

Comment:
Should we have this file as part of the complete solution? Or should we delete it?